tests/unit/mixins/ajax-request-test.js

```diff
@@ -906,6 +906,8 @@ describe('Unit | Mixin | ajax request', function() {
           .catch(function(reason) {
             expect(reason instanceof errorClass).to.be.ok;
             expect(reason.errors && typeOf(reason.errors) === 'array').to.be.ok;
+            expect(reason.errors[0].id).to.equal(1);
+            expect(reason.errors[0].message).to.equal('error description');
```

Comment:
If you'd like this logic elsewhere, that's fine. But I figured since you already had an assertion that the `reason.errors` is an array, i thought i'd add to that asserting that the content is correct.